.rubocop.yml

```diff
@@ -32,5 +32,6 @@ Rails/LexicallyScopedActionFilter:
   Exclude:
     - app/controllers/api/v1/api_controller.rb
 
-Style/EndOfLine:
-  Enabled: false
\ No newline at end of file
+Layout/EndOfLine:
+  Exclude:
+    -  spec\models\target_spec.rb
```

Comment:
looks like you dont need this anymore 🌚

Reply:
But if they aren't there, I have this offense: 
'spec/models/target_spec.rb:1:1: C: Layout/EndOfLine: Carriage return character missing. # == Schema Information ...' and I don't know why.